src/widgetastic/widget.py

```diff
@@ -1805,11 +1815,13 @@ def row_by_cell_or_widget_value(self, column, key):
         except RowNotFound:
             for row in self.rows():
                 if row[column].widget is None:
-                    continue
+                    continue  # the column has a widget...
                 if not row[column].widget.is_displayed:
-                    continue
+                    continue  # the column widget is displayed...
```

Comment:
*REQUIRED*
`if not row[column].widget.is_displayed` should mean your comment says `# the column widget is NOT displayed...`

Reply:
ah yeah these ended up being confusing. they really apply to the next check, not the reason we're continuing. Will update.